### PR TITLE
Add IPv6 Support

### PR DESCRIPTION
**Pull request by davebv**
I modified your script a little to provide ipv6 support

### CODE REVIEW

**Comment by davebv** [comment]
Is there a possibility to merge this one into your master branch?

File: duck.sh [hunk 1]
```diff
@@ -42,17 +51,31 @@ if [[ "${INTERVAL: -1}" == 'm' && "${INTERVAL:0:-1}" -lt 5 ]]; then
...
+endpoint=https://www.duckdns.org/update
+
 #-----------------------------------------------------------------------------------------------------------------------
 
 function ts {
   echo [`date '+%b %d %X'`]
 }
 
 #-----------------------------------------------------------------------------------------------------------------------
+sleep 2
```
Comment (coppit) [inline duck.sh]: Why the sleep 2?

File: README.md [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
 
 To check the status, run `docker logs duckdns`.
+
+## IPv6
+
+By default IPv6 is not enabled. In order to enable it, uncomment the line in duck.conf to enable IPv6
+`IPV6=yes`
+
+## Credits
+Image updated from https://github.com/coppit/docker-duckdns
```
Comment (coppit) [inline README.md:26]: Changed to this in my branch:

IPv6 support by [davebv](https://github.com/davebv).

File: README.md [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
 This is a simple Docker container for running the [Duck DNS](http://duckdns.org) dynamic DNS update script. It will keep your domain.duckdns.org DNS alias up-to-date as your home IP changes. The script runs every 5 minutes.
 
-Usage
------
+Both IPv4 and IPv6 are supported.
+
+## Usage
 
-This docker image is available as a [trusted build on the docker index](https://index.docker.io/u/coppit/duckdns/).
+This docker image is available as a [trusted build on the docker index](https://index.docker.io/u/davebv/docker-duckdns/).
```
Comment (coppit) [inline README.md:10]: Reverted these changes in my branch

File: duck.sh (hunk 1 above)
Comment (davebv) [reply duck.sh]: Do not remember why I wrote it. Cannot make any sense about it now... could be removed?

File: README.md (hunk 3 above)
Comment (davebv) [reply README.md:10]: looks good

File: README.md (hunk 2 above)
Comment (davebv) [reply README.md:26]: looks good too

**Comment by coppit** [comment]
I merged it in, with edits.

**Pull request closed without merging**